README.md

```diff
@@ -134,7 +138,7 @@ model.getJobs(config)
 | :-------------   | :---- | :--- | :--- | :-------------|
 | config        | Object | No | | Configuration Object |
 | config.params | Object | No | | Fields to search on |
-| config.params.sort | Boolean | No | false| Sorting by createTime |
+| config.params.sort | Boolean | No | false | Sorting by `createTime` |
```

Comment:
Interesting. I don't see this

Reply:
yeah I'm not sure where it comes from
I can remove it